sdk/openai/openai/package.json

```diff
@@ -105,7 +105,7 @@
     "@azure-tools/test-recorder": "^3.0.0",
     "@azure-tools/test-credential": "^1.0.0",
     "@azure/test-utils": "^1.0.0",
-    "@azure/core-rest-pipeline": "^1.12.0",
+    "@azure/core-rest-pipeline": "^1.12.2",
```

Comment:
Could we have only one entry for core-rest-pipeline and under `dependencies` not under `devDependencies`?